src/modules/Security.rb

```diff
@@ -291,46 +293,19 @@ def ReadServiceSettings
       nil
     end
 
+    def inittab_shutdown_configured?
+      inittab = SCR.Dir(path(".etc.inittab"))
+      inittab.include? "ca"
```

Comment:
Yes, they are optional, but the [style guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md#parameters) allows omitting them only in special cases.